src/yunohost/backup.py

```diff
@@ -1776,8 +1776,13 @@ def mount(self, restore_manager):
         tar.close()
 
         # Mount the tarball
-        ret = subprocess.call(['archivemount', '-o', 'readonly',
-                               self._archive_file, self.work_dir])
+        try:
+            ret = subprocess.call(['archivemount', '-o', 'readonly',
```

Comment:
Unfortunately no, and that's where the bug comes from : it raises an exception in that case with a message such as "Command or file not found". C.f. the original Redmine issue